projects/storefrontlib/shared/components/captcha/captcha-api-config.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable, Type } from '@angular/core';
+import { Config } from '@spartacus/core';
+import { CaptchaProvider } from './captcha.model';
+
+@Injectable({
+  providedIn: 'root',
+  useExisting: Config,
+})
+export abstract class CaptchaApiConfig {
```

Comment:
We should add doc comments to these since they will be user facing.

Reply:
Added

Reply:
I mean for the properties too. For example, its not really clear what some might use `fields` for.